cpp/src/las/header.cpp

```diff
@@ -10,6 +10,22 @@
 
 namespace copc::las
 {
+
+LasHeader::LasHeader(const LasHeader &header, int8_t point_format_id, uint16_t point_record_length,
```

Comment:
I think we should make a general copy constructor and then just change the four parameters that changed, instead of the other way around, does that make sense?

Something like:

```c++
LasHeader::LasHeader(const LasHeader &header) : LasHeader(header.scale, header.offset ...

LasHeader::LasHeader(const LasHeader &header, int8_t point_format_id, uint16_t point_record_length, const Vector3 &scale, const Vector3 &offset) : LasHeader(header)
{
	this->point_format_id = point_format_id;
	this->scale = scale; 
	...
}
+++

Reply:
Lol yes that would make a lot more sense :D